.gitignore

```diff
@@ -36,8 +36,10 @@ dist
 
 # workspace specific
 site/blocks-data.json
+site/playwright-report
+site/test-results
 site/public/blocks
 site/public/schemas
 site/site-map.json
 
-.parcel-cache
```

Comment:
I could  not find any signs of using parcel in the repo, so we should not need this line